validation-test/Runtime/Inputs/class-layout-from-objc/Classes.swift

```diff
@@ -0,0 +1,40 @@
+import Resilient
+import Foundation
+
+public class StaticClass: NSObject {
+  @objc var first: Int = 0
+  var forceUnalignment: Int8 = 0
+  var middle = GrowsToInt64()
+  @objc var last: Int = 0
+
+  @objc public static var totalSize: Int {
+    return (2 * MemoryLayout<Int>.size) +
+           (2 * MemoryLayout<GrowsToInt64>.size) + // alignment
+           MemoryLayout<Int>.size
+  }
+}
+
+public class DynamicClass: NSObject {
```

Comment:
Can you also test a subclass of an Objective-C class that is not NSObject (and has ivars)? We've had bugs with ivar layout in this case in the past.

Reply:
Good idea, will add.